Makefile

```diff
@@ -461,21 +461,7 @@ check-docker-hashes-consistency: $(DOCKERFILE_FROM_CHECKER)
 	@echo "Checking Dockerfiles for inconsistencies"
 	$(DOCKERFILE_FROM_CHECKER) ./ \
 		-i ./eve-tools/bpftrace-compiler/Dockerfile \
-		-i ./eve-tools/bpftrace-compiler/root/Dockerfile \
-		-i pkg/bsp-imx/Dockerfile \
-		-i pkg/vtpm/Dockerfile \
-		-i pkg/optee-os/Dockerfile \
-		-i pkg/installer/Dockerfile \
-		-i pkg/wwan/Dockerfile \
-		-i pkg/wlan/Dockerfile \
-		-i pkg/watchdog/Dockerfile \
-		-i pkg/uefi/Dockerfile \
-		-i pkg/acrn/Dockerfile \
-		-i pkg/acrn-kernel/Dockerfile \
-		-i pkg/u-boot/Dockerfile \
-		-i pkg/udev/Dockerfile \
-		-i pkg/xen-tools/Dockerfile \
-		-i pkg/xen/Dockerfile
+		-i pkg/installer/Dockerfile
```

Comment:
Should it be rebased to use the new variables introduced by @deitch, I guess?

Reply:
Yup. And I just opened a PR based on that to bump the alpines as well, but _remove_ `pkg/installer` from the exceptions list 😄 

Not sure if that helps, hinders or is neutral to this one. Feel free to throw mine away if needed.

Reply:
I think it's best to wait for rebase until https://github.com/lf-edge/eve/pull/4348 is merged ...

Reply:
> I think it's best to wait for rebase until #4348 is merged ...

And that PR is dependent on https://github.com/lf-edge/eve/pull/4350